adClickFlow/shared/testCases.json

```diff
@@ -211,6 +220,29 @@
               "url": "https://www.ad-company.site/ping.gif",
               "status": "parent blocked"
             }
+          ],
+          "pixels": [
+            {
+              "name": "m_ad_click_detected",
+              "params": {
+                "appVersion": "APP_VERSION",
+                "domainDetection": "heuristic_only",
+                "heuristicDetectionEnabled": "1",
+                "domainDetectionEnabled": "1"
+              }
+            },
+            {
+              "name": "m_ad_click_active",
+              "params": {
+                "appVersion": "APP_VERSION"
+              }
+            },
+            {
+              "name": "m_pageloads_with_ad_attribution",
```

Comment:
There is an assumption here that for the test runner that we force this to fire at the end of each test case. I think that's okay, but just checking that it's expected.

Reply:
Yea, that's right. I would have mentioned that in a comment (like I did in the other version of this file) but unfortunately JSON doesn't support comments.